picard/ui/options/plugins.py

```diff
@@ -86,6 +86,8 @@ def __init__(self, parent=None):
         super().__init__(parent)
         self.ui = Ui_PluginsOptionsPage()
         self.ui.setupUi(self)
+        #fix for PICARD-1226, QT bug around
```

Comment:
Can you just add links to related QT bugs for future reference ?

Reply:
Ok i will add one.. :-)